src/Install.php

```diff
@@ -448,16 +449,28 @@ public static function create_events() {
 		if ( ! wp_next_scheduled( 'wc_admin_daily' ) ) {
 			wp_schedule_event( time(), 'daily', 'wc_admin_daily' );
 		}
-		// @todo This is potentially redundant when the core package exists.
+		// Note: this is potentially redundant when the core package exists.
 		wp_schedule_single_event( time() + 10, 'generate_category_lookup_table' );
 	}
 
+	/**
+	 * Delete obsolete notes.
+	 */
+	protected static function delete_obsolete_notes() {
+		$obsolete_notes_names = array(
```

Comment:
Do you think there is any point in adding a filter here for other packages/plugins to hook into?

Reply:
Yeah, that would be a nice addition. I'll add a filter to this PR.

Reply:
Added filter.